src/util/magic.h

```diff
@@ -16,4 +16,8 @@ const std::size_t MAX_CONCRETE_STRING_SIZE = 1 << 26;
 // The top end of the range of integers for which dstrings are precomputed
 constexpr std::size_t DSTRING_NUMBERS_MAX = 64;
 
+/// Limit the size of arrays for which field_sensitivity gets applied.
+/// Necessary because large constant arrays slow-down the process.
+constexpr unsigned max_field_sensitive_array_size = 64;
```

Comment:
Any reason why this is `unsigned` and not `size_t`?

Reply:
Any reason why this is lower case and not all-caps as all the other constants defined in this file?